src/vonage/video.py

```diff
@@ -65,52 +60,49 @@ def get_stream(self, session_id, stream_id):
         return self._client.get(
             self._client.video_host(),
             f'/v2/project/{self._client.application_id}/session/{session_id}/stream/{stream_id}',
-            auth_type=Video.auth_type
+            auth_type=Video.auth_type,
         )
 
     def list_streams(self, session_id):
         return self._client.get(
             self._client.video_host(),
             f'/v2/project/{self._client.application_id}/session/{session_id}/stream',
-            auth_type=Video.auth_type
+            auth_type=Video.auth_type,
         )
 
     def set_stream_layout(self, session_id, items):
         return self._client.put(
             self._client.video_host(),
             f'/v2/project/{self._client.application_id}/session/{session_id}/stream',
             items,
-            auth_type=Video.auth_type
+            auth_type=Video.auth_type,
         )
 
     def send_signal(self, session_id, type, data, connection_id=None):
         if connection_id:
-            request_uri = f'/v2/project/{self._client.application_id}/session/{session_id}/connection/{connection_id}/signal'
+            request_uri = (
+                f'/v2/project/{self._client.application_id}/session/{session_id}/connection/{connection_id}/signal'
+            )
         else:
             request_uri = f'/v2/project/{self._client.application_id}/session/{session_id}/signal'
-        
+
         params = {'type': type, 'data': data}
 
-        return self._client.post(
-            self._client.video_host(),
-            request_uri,
-            params,
-            auth_type=Video.auth_type
-        )
+        return self._client.post(self._client.video_host(), request_uri, params, auth_type=Video.auth_type)
 
     def disconnect_client(self, session_id, connection_id):
         return self._client.delete(
             self._client.video_host(),
             f'/v2/project/{self._client.application_id}/session/{session_id}/connection/{connection_id}',
-            auth_type=Video.auth_type
+            auth_type=Video.auth_type,
         )
 
-    def mute_stream(self, session_id, stream_id):        
+    def mute_stream(self, session_id, stream_id):
         return self._client.post(
             self._client.video_host(),
             f'/v2/project/{self._client.application_id}/session/{session_id}/stream/{stream_id}/mute',
-            params=None,           
-            auth_type=Video.auth_type
+            params=None,
+            auth_type=Video.auth_type,
```

Comment:
What's with the commas?

Reply:
I'm formatting the new stuff with Black: https://black.readthedocs.io/en/stable/ and it's a default. I'm going to do the rest of the SDK asap as well.